src/features/pilot_management/PilotSheet/sections/mech/components/MechNav.vue

```diff
@@ -3,18 +3,24 @@
     <div id="cap" />
     <div v-if="$vuetify.breakpoint.mdAndUp" class="d-inline">
       <cc-nav-item tile depressed :selected="selected === 0" @click="$emit('set-page', 0)">
-        MECH CONFIGURATION
+        <cc-tooltip inlinde delayed content="Active Mech Configuration">
+          MECH CONFIGURATION
+        </cc-tooltip>
```

Comment:
This is nitpicky and I know it, but for consistency sake can you wrap the cc-nav-item in the cc-tooltip as well?
This is how it is laid out for "COMBAT ANALYTICS" on line 10.

Reply:
Yup, I can do that. I was basing my changes off PlayerNav.vue, which does it differently, but I can nest it outside for consistency in the file.